src/Diagnostics.AIProjects/SearchAPI/RegistryReader.py

```diff
@@ -0,0 +1,16 @@
+from winreg import *
+githubFolderPath = ""
+kustoClientId = ""
+kustoClientSecret = ""
+try:
+	aReg = ConnectRegistry(None,HKEY_LOCAL_MACHINE)
+	aKey = OpenKey(aReg, r"SOFTWARE\Microsoft\IIS Extensions\Web Hosting Framework")
+	sourceWatcherKey = OpenKey(aKey, "SourceWatcher")
+	dataProvidersKey = OpenKey(aKey, "DiagnosticDataProviders")
+	githubKey = OpenKey(sourceWatcherKey,"Github")
```

Comment:
Will this fail if GitHub key is not present.
We dont add Github Registry if the SourceWatcher Mode is set to Local instead of Github

Reply:
its fine for now to not support local mode.. but add a backlog item for that for tracking

Reply:
Done.